scss/_patterns_chip.scss

```diff
@@ -82,55 +80,145 @@
   @include vf-chip-caution;
   @include vf-chip-negative;
   @include vf-chip-information;
+
+  // Theming
+  @if ($theme-default-p-chip == 'dark') {
+    .p-chip {
```

Comment:
This part seems extremely repetitive, but I'm not sure if we can do anything about it - we do need separate color treatment for each of those variants…

Reply:
I had the same feeling when writing it but couldn't figure out a way around it!